packages/rocketchat-mongo-config/server/index.js

```diff
@@ -3,6 +3,20 @@ import { PassThrough } from 'stream';
 
 import { EmailTest } from 'meteor/email';
 import { Mongo } from 'meteor/mongo';
+import { HTTP } from 'meteor/http';
+
+// Set default HTTP call timeout to 20s
+const envTimeout = parseInt(process.env.HTTP_DEFAULT_TIMEOUT, 10);
+const timeout = !isNaN(envTimeout) ? envTimeout : 20000;
+
+const { call } = HTTP;
+HTTP.call = function _call(method, url, options = {}, callback) {
+	if (!options.timeout) {
+		options.timeout = timeout;
```

Comment:
I think we should avoid changing params objects, as they're always a reference and changing them can lead to very hard to track bugs.

in this case I'd suggest something like:
```
const defaultTimeout = options.timeout ? options : { ...options, timeout };
call.call(HTTP, method, url, defaultTimeout, callback);
```

I don't want to block the merge so I'm sending this as a comment so we can discuss about it. wdyt?